README.md

```diff
@@ -9,7 +9,7 @@ go wrapper for [chartjs](http://chartjs.org)
 
 Chartjs charts are defined purely in JSON, so this library is mostly
 structs and struct-tags that dictate how to marshal to JSON. None of the currently
-implemented parts are stringly-typed in this library so it can avoid many errors.
+implemented parts are strongly-typed in this library so it can avoid many errors.
```

Comment:
this was intentional. but you could change it to "All of the currently implemented parts are strongly-type..."

Reply:
I mean if you meant stringly then that's cool, just I was unaware that stringly was 1. A word or 2. Applicable in this context